modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/web/internal/portlet/action/test/BoundObjectDefinitionsExportImportTest.java

```diff
@@ -0,0 +1,283 @@
+@FeatureFlags("LPS-187142")
+@RunWith(Arquillian.class)
+public class BoundObjectDefinitionsExportImportTest
+	extends BaseExportImportTestCase {
+
+	@ClassRule
+	@Rule
+	public static final LiferayIntegrationTestRule liferayIntegrationTestRule =
+		new LiferayIntegrationTestRule();
+
+	@Before
+	public void setUp() throws Exception {
+		user = TestPropsValues.getUser();
+
+		ObjectDefinitionResource.Builder builder =
+			_objectDefinitionResourceFactory.create();
+
+		_objectDefinitionResource = builder.user(
+			user
+		).build();
+
+		Class<?> clazz = getClazz();
+
+		_defaultObjectDefinitionJSONString = StringUtil.read(
+			clazz.getResourceAsStream(
+				"dependencies/test-default-object-definition-draft.json"));
+		_defaultObjectRelationshipJSONString = StringUtil.read(
+			clazz.getResourceAsStream(
+				"dependencies/test-default-object-relationship.json"));
+	}
+
+	@Test
+	public void testExportImportBoundObjectDefinitions() throws Exception {
+
+		// Import bound object definitions as draft
+
+		String draftObjectDefinitionsJSONString = String.valueOf(
+			JSONUtil.putAll(
+				_jsonFactory.createJSONObject(
+					_defaultObjectDefinitionJSONString
+				).put(
+					"externalReferenceCode", "TESTOBJECTDEFINITION1"
+				).put(
+					"name", "TestObjectDefinition1"
+				).put(
+					"objectRelationships",
+					JSONUtil.put(
+						_createOneToManyObjectRelationship(
+							"TESTOBJECTDEFINITION1", "TESTOBJECTDEFINITION2",
+							"TestObjectDefinition2", "objectRelationship1"))
+				).put(
+					"rootObjectDefinitionExternalReferenceCode",
+					"TESTOBJECTDEFINITION1"
+				),
+				_jsonFactory.createJSONObject(
+					_defaultObjectDefinitionJSONString
+				).put(
+					"externalReferenceCode", "TESTOBJECTDEFINITION2"
+				).put(
+					"name", "TestObjectDefinition2"
+				).put(
+					"objectRelationships",
+					JSONUtil.put(
+						_createOneToManyObjectRelationship(
+							"TESTOBJECTDEFINITION2", "TESTOBJECTDEFINITION3",
+							"TestObjectDefinition3", "objectRelationship2"))
+				).put(
+					"rootObjectDefinitionExternalReferenceCode",
+					"TESTOBJECTDEFINITION1"
+				),
+				_jsonFactory.createJSONObject(
+					_defaultObjectDefinitionJSONString
+				).put(
+					"externalReferenceCode", "TESTOBJECTDEFINITION3"
+				).put(
+					"name", "TestObjectDefinition3"
+				).put(
+					"rootObjectDefinitionExternalReferenceCode",
+					"TESTOBJECTDEFINITION1"
+				)));
+
+		JSONArray expectedObjectDefinitionsJSONArray =
+			_jsonFactory.createJSONArray(draftObjectDefinitionsJSONString);
+
+		for (int i = 1; i <= 2; i++) {
+			JSONObject objectDefinitionJSONObject =
+				expectedObjectDefinitionsJSONArray.getJSONObject(i);
+
+			JSONArray objectFieldsJSONArray =
+				objectDefinitionJSONObject.getJSONArray("objectFields");
+
+			objectFieldsJSONArray.put(
+				_jsonFactory.createJSONObject(
+				).put(
+					"name",
+					StringBundler.concat(
+						"r_objectRelationship", i, "_c_testObjectDefinition", i,
+						"Id")
+				));
+		}
```

Comment:
I noticed that doing this adds more complexity and "distraction" to the test, because we will need to have many more lines of code just to build the complete json relationship object fields and add them to the `objectFields` array in each bound object definition separately.

The way it is, we can use just the name of the field, for example. And it is a lot simpler to add.

It's also possible to do them both separately instead of in a for loop, if that's clearer